android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java

```diff
@@ -49,11 +49,15 @@ public void asyncRet(String param, JSCallback callback) {
     @JSMethod(uiThread = true)
     public void show(String params) throws JSONException {
         JSONObject json = new JSONObject();
-        try { json = new JSONObject(params); } catch (Throwable t) { Log.e("-> show", "Could not parse malformed JSON: \"" + json + "\""); }
```

Comment:
@phanisaripalli is there any way we can refactor this to become a method and call it from both methods that are using this parsing?

Reply:
yea, sure. let me commit.

Reply:
@dseeker please have a look